meteor/client/ui/ClockView/ClockView.tsx

```diff
@@ -332,10 +25,19 @@ interface IStateHeader {}
 export const ClockView = withTranslation()(
 	withTracker(function(props: IPropsHeader) {
 		let studioId = objectPathGet(props, 'match.params.studioId')
-		const playlist = RundownPlaylists.findOne({
-			active: true,
-			studioId: studioId,
-		})
+		const playlist = RundownPlaylists.findOne(
+			{
+				active: true,
+				studioId: studioId,
+			},
+			{
+				fields: {
+					_id: 1,
```

Comment:
Think I'd like a comment that explains why id === 1 is always the right choice here, it's not immediately understandable to me :)

Reply:
This was put there in order to limit the reactivity, but I now actually realize, that we need this to be pretty reactive, and the RundownPlaylist object isn't all that reactive.